dist/platforms/ubuntu/steps/activate.sh

```diff
@@ -28,10 +28,14 @@ if [[ -n "$UNITY_LICENSE" ]] || [[ -n "$UNITY_LICENSE_FILE" ]]; then
   fi
 
   # Activate license
-  ACTIVATION_OUTPUT=$(unity-editor \
-      -logFile /dev/stdout \
-      -quit \
-      -manualLicenseFile $FILE_PATH)
+  # Set to always return true, otherwise the shell will exit on pipelines set to fail on error
+  ACTIVATION_OUTPUT=UnityActivation.log
+
+  unity-editor \
+    -logFile "$ACTIVATION_OUTPUT" \
+    -quit \
+    -manualLicenseFile $FILE_PATH \
+    || true
```

Comment:
Note that unity-builder is made specifically for running on GitHub Actions but I'm assuming you know that.

That would work, but personally I wouldn't pollute our code with the assumption that the editor is always going to return exit code 1 (bug in unity). Perhaps in some versions the exit code starts working.

Therefor I believe a comment like on line 31 is better in case of `unity-builder` specifically.